README.md

```diff
@@ -1 +1 @@
-testing semantic versioning 5.0.0
```

Comment:
The code change from testing semantic versioning 5.0.0 to testing semantic versioning 5.2.0 seems correct as it indicates an update in the version being tested.